README.md

```
@@ -123,19 +128,32 @@ This project is licensed under the terms of the MIT license.
 - **Discord Bot Notifications**: Get real-time alerts via Discord Bot for:
-  - Trade executions (buy/sell)
+  - Infinite buying executions (buy/sell with detailed stats)
+  - Manual trade executions
   - Order cancellations
-  - Error notifications
+  - Error notifications and cycle completions
   - System status updates
 - **Rich Embed Messages**: Beautiful formatted messages with colors and fields
+- **Detailed Trading Info**: Shows current round, average price, total investment, target price
+
+## API Endpoints
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)